File: Aerolinea/web/js/Viaje.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
function Viaje(numero_viaje,fecha,avion,vuelo){
	this.Viaje(numero_viaje,fecha,avion,vuelo);
}
Viaje.prototype={
        numero_viaje: 0,
        fecha:"",
        avion:"",
        vuelo: 0,
	Viaje:function(numero_viaje,fecha,avion,vuelo){
		this.numero_viaje=numero_viaje;
                this.fecha=fecha;
		this.avion=avion;
		this.vuelo=vuelo;
	},
        toString:function(){
	  return this.numero_viaje;
	}
}
Viaje.from=function(plain){
    viaje = new Viaje(plain.numero_viaje,plain.fecha,plain.avion,plain.vuelo);
    return viaje;
};

Viaje.to= function(viaje){
    return{
         _class : 'Viaje',
         numero_viaje: viaje.numero_viaje,
         fecha: viaje.fecha,
         avion: viaje.avion,
         vuelo: viaje.vuelo
    };
};

File: Aerolinea/src/java/airline/model/Viaje.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package airline.model;
import java.io.Serializable;
/**
 *
 * @author AndreyCh
 */
public class Viaje implements Serializable, Jsonable{
    int numero_viaje;
    Avion avion;
    Vuelo vuelo;

    public Viaje(int numero_viaje, Avion avion, Vuelo vuelo) {
        this.numero_viaje = numero_viaje;
        this.avion = avion;
        this.vuelo = vuelo;
    }

    public Viaje() {
    }

    public int getNumero_viaje() {
        return numero_viaje;
    }

    public Avion getAvion() {
        return avion;
    }

    public Vuelo getVuelo() {
        return vuelo;
    }

    public void setNumero_viaje(int numero_viaje) {
        this.numero_viaje = numero_viaje;
    }

    public void setAvion(Avion avion) {
        this.avion = avion;
    }

    public void setVuelo(Vuelo vuelo) {
        this.vuelo = vuelo;
    }
    
}
